packages/overlays/stories/index.stories.mdx

```diff
@@ -0,0 +1,646 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { renderLitAsNode } from '@lion/core';
+import {
+  OverlayMixin,
+  withBottomSheetConfig,
+  withDropdownConfig,
+  withModalDialogConfig,
+} from '../index.js';
+
+import './lion-demo-overlay.js';
+import './applyDemoOverlayStyles.js';
+
+<Meta title="Overlay System/System" parameters={{ component: 'lion-demo-overlay' }} />
+
+# Overlays System
+
+Supports different types of overlays like dialogs, toasts, tooltips, dropdown, etc.
+
+Manages their position on the screen relative to other elements, including other overlays.
+
+`lion-dialog` can be used to make it easy to use our Overlay System declaratively. It is a simple Web Component that implements the `OverlayMixin`.
+Configuration options can be easily overrided, and the component can be extended where needed, to override event listeners and more.
+
+In our demos, we will use a quick demo web component `lion-demo-overlay` to showcase the features of the Overlay System.
+
+> Again, we recommend using `lion-dialog` as a starting point for using the Overlay System declaratively with a Web Component,
+> but for our demos we don't want a dependency on that package.
+
+<Story name="default">
+  {html`
+    <lion-demo-overlay>
+      <button slot="invoker">Click me to open the overlay!</button>
+      <div slot="content" class="demo-overlay">
+        Hello! You can close this notification here:
+        <button
+          class="close-button"
+          @click=${e => e.target.dispatchEvent(new Event('close-overlay', { bubbles: true }))}
+        >
+          ⨯
+        </button>
+      </div>
+    </lion-demo-overlay>
+  `}
+</Story>
+
+```html
+<lion-demo-overlay>
+  <button slot="invoker">Click me to open the overlay!</button>
+  <div slot="content" class="demo-overlay">
+    Hello! You can close this notification here:
+    <button
+      class="close-button"
+      @click=${e => e.target.dispatchEvent(new Event('close-overlay', { bubbles: true }))}
+    >
+      ⨯
+    </button>
+  </div>
+</lion-demo-overlay>
+```
+
+## Features
+
+- **OverlaysManager**, a global repository keeping track of all different types of active overlays
+- **OverlayController**, a single controller class for handling an overlay
+- **OverlayMixin**, a mixin that can be used to create Web Components that use the OverlayController under the hood
+
+## How to use
+
+### Installation
+
+```sh
+npm i --save @lion/overlays
+```
+
+This package should only be used by **subclassers** who want to either:
```

Comment:
It would be nice to have a reference to our docs that explain the difference between Sub classers and  Application Developers

Reply:
outside scope of this story: will pick up later